include/clangmetatool/collectors/find_calls_data.h

```diff
@@ -9,22 +9,45 @@
 namespace clangmetatool {
   namespace collectors {
 
-  struct FindCallsData {
-    std::map<const clang::FunctionDecl*,
-             const clang::CallExpr*> call_context;
-    std::map<const clang::CallExpr*,
-             const clang::DeclRefExpr*> call_ref;
-    std::map<const clang::CallExpr*,
-             const clang::DeclRefExpr*> call_argref;
-    std::map<const clang::CallExpr*,
-             const clang::StringLiteral*> call_argstr;
-  };
+    /**
+     * The data collected by the FindCalls collector
+     */
+    struct FindCallsData {
+
+      /**
+       * The declaration of the function and the context in which it occurs
+       */
+      std::map<const clang::FunctionDecl*,
+               const clang::CallExpr*> call_context;
```

Comment:
I think this needs to be a `std::multimap`, no?

Reply:
yes, you're right